### PR TITLE
Proofreading pass

### PR DESCRIPTION
**Pull request by Buzzardo**
I proofread the documentation and made it more formal.

### CODE REVIEW

File: docs/src/main/asciidoc/migrations.adoc [hunk 1]
```diff
@@ -4,33 +4,38 @@
 
 == Migrations
 
-In the following document we will write about necessary migration steps between versions.
+This section covers from one version of Spring Cloud Contract Verifier to the next
```
Comment (marcingrzejszczak) [inline docs/src/main/asciidoc/migrations.adoc]: This section covers *upgrading* / *migrating* ?

File: docs/src/main/asciidoc/migrations.adoc [hunk 2]
```diff
@@ -111,32 +116,42 @@ task copyStubs(type: Copy, dependsOn: 'generateWireMockClientStubs') {
...
-https://github.com/spring-cloud/spring-cloud-contract/issues/260[issue 260].
+See https://github.com/spring-cloud/spring-cloud-contract/issues/260[issue 260] for more
+detail.
 
 ==== New methods in TemplateProcessor
 
-In order to add support for `fromRequest.path` some methods had to be added to the
+In order to add support for `fromRequest.path`, some methods had to be added to the
 `TemplateProcessor` interface.
+// TODO What methods?
```
Comment (marcingrzejszczak) [inline docs/src/main/asciidoc/migrations.adoc]: Ooops... :P

the missing methods are `path()` and `path(int index)`

File: docs/src/main/asciidoc/spring-cloud-contract.adoc [hunk 3]
```diff
@@ -10,21 +10,22 @@
 = Spring Cloud Contract
 
 _Documentation Authors: Adam Dudczak, Mathias Düsterhöft, Marcin Grzejszczak, Dennis Kieselhorst, Jakub Kubryński, Karol Lassak,
-Olga Maciaszek-Sharma, Mariusz Smykuła, Dave Syer_
+Olga Maciaszek-Sharma, Mariusz Smykuła, Dave Syer_, Jay Bryant
```
Comment (marcingrzejszczak) [inline docs/src/main/asciidoc/spring-cloud-contract.adoc]: :+1: :+1: :+1: :+1: :+1: :+1: 

I tried to keep the names in alphabetical order. WDYT?

File: docs/src/main/asciidoc/verifier_messaging.adoc [hunk 4]
```diff
@@ -151,12 +154,13 @@ And the following Spock test would be created:
...
+=== Consumer Stub Generation
 
-Unlike the HTTP part - in Messaging we need to publish the Groovy DSL inside the JAR with a stub. Then it's parsed on the consumer side
-and proper stubbed routes are created.
+Unlike the HTTP part, in messaging, we need to publish the Groovy DSL inside the JAR with
+a stub. Then it is parsed on the consumer side and proper stubbed routes are created.
 
-For more information please consult the Stub Runner Messaging sections.
+For more information, please consult the Stub Runner Messaging sections.
+// TODO Add a link to those sections.
```
Comment (marcingrzejszczak) [inline docs/src/main/asciidoc/verifier_messaging.adoc]: https://cloud.spring.io/spring-cloud-contract/spring-cloud-contract.html/#stub-runner-for-messaging

File: docs/src/main/asciidoc/migrations.adoc (hunk 1 above)
Comment (Buzzardo) [reply docs/src/main/asciidoc/migrations.adoc]: Thank you. I was apparently typing faster than I was thinking. I'll fix it.

File: docs/src/main/asciidoc/migrations.adoc (hunk 2 above)
Comment (Buzzardo) [reply docs/src/main/asciidoc/migrations.adoc]: Thank you. I'll add that information.

File: docs/src/main/asciidoc/verifier_messaging.adoc (hunk 4 above)
Comment (Buzzardo) [reply docs/src/main/asciidoc/verifier_messaging.adoc]: Thank you. I'll add that.

File: docs/src/main/asciidoc/spring-cloud-contract.adoc (hunk 3 above)
Comment (Buzzardo) [reply docs/src/main/asciidoc/spring-cloud-contract.adoc]: I think only a few people look at the list of authors. That said, I think at least some people think the primary author goes first, followed by whomever made the next biggest contribution, and so on. Looked at that way, you should be first, and I should be last. If I alphabetize, people may think I'm the primary author, which would be a false impression. I just swept the floors. You built the house.

I'll hold off on putting my name first, until you've had a chance to consider the issue.

Comment (marcingrzejszczak) [reply docs/src/main/asciidoc/spring-cloud-contract.adoc]: You're the docs master so I trust your judgment :)

Comment (Buzzardo) [reply docs/src/main/asciidoc/spring-cloud-contract.adoc]: Then I'll leave my name at the end, both as a matter of humility and to reflect reality. :)

Thanks, Marcin.

**Comment by marcingrzejszczak** [comment]
@Buzzardo awesome work, thanks a lot!

**Pull request merged**